tests/flow/test_foreach.py

```
@@ -0,0 +1,710 @@
+from common import *
+from collections import Counter
```

Ruff (0.12.2)

`collections.Counter` imported but unused

Remove unused import: `collections.Counter`

(F401)